src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -580,6 +581,17 @@ public void setServerURL(String serverURL) {
         mConfig.setServerURL(serverURL);
     }
 
+    /**
+     * Set the base URL used for Mixpanel API requests.
+     * Useful if you need to proxy Mixpanel requests. Defaults to https://api.mixpanel.com.
+     * To route data to Mixpanel's EU servers, set to https://api-eu.mixpanel.com
+     *
+     * @param serverURL the base URL used for Mixpanel API requests
+     * @param callback the callback for mixpanel proxy server api headers and status
+     */
+    public void setServerURL(String serverURL, MixpanelServerCallback callback) {
```

Comment:
missing the definition of `MixpanelServerCallback`, are you meant to use `ProxyServerInteractor`?

Reply:
yes. updated it